src/typescript/azure-openapi-validator/tests/composite-azure-test.ts

```diff
@@ -1,22 +0,0 @@
-/*---------------------------------------------------------------------------------------------
- *  Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Maybe not delete this file altogether. Can we at least add a dummy test case so it becomes a placeholder for any new composite tests we would like to add later

Reply:
Is there a reason to have this file around other than as a dummy?

Reply:
Not that I can think of.

Reply:
In that case I guess it comes down to what your team's policies are - if you'll accept a dummy file here, I'll totally add it back in.

Reply:
Delete this for now, guess we can check it from this commit letter.